ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.js

```diff
@@ -0,0 +1,49 @@
+import { SyncedCron } from 'meteor/littledata:synced-cron';
+
+import { settings } from '../../../../../app/settings/server';
+import { LivechatRooms } from '../../../../../app/models/server';
+import { Livechat } from '../../../../../app/livechat/server/lib/Livechat';
+
+export class VisitorInactivityMonitor {
+	start() {
```

Comment:
We need to create a `stop` method as well, as we do here:
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/statistics/LivechatAgentActivityMonitor.js#L25-L33